modules/csharp/examples/code_snippets/Program.cs

```diff
@@ -449,6 +449,15 @@ private static void CloseDatabase()
             // end::close-database[]
         }
 
+        private static void DatabaseFullsync()
+        {
+           // tag::database-fullsync[] 
+          var config = new DatabaseConfiguration();
+          // this enables fullsync
+          config.FullSync = false;
```

Comment:
Misleading comment.  `false` is not going to enable anything.

Reply:
Good spot, changed parameter to true.